File: ControlRemote/Web/ClientApp/src/app/Dto/ActionSortByDateTimeModel.ts
import { ActionSortByHourTimeModel } from "./ActionSortByHourTimeModel";

export class ActionSortByDateTimeModel {
    dateTimeAction: Date;
    commands: ActionSortByHourTimeModel[];

    public constructor(_dateTimeAction: Date, _commands: ActionSortByHourTimeModel[]) {
        this.dateTimeAction = _dateTimeAction;
        this.commands = _commands;
    }
}

File: ControlRemote/Web/Dto/ActionSortByDateTimeModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Web.Dto
{
    public class ActionSortByDateTimeModel
    {
        public DateTime DateTimeAction { get; set; }
        public List<ActionSortByHourTimeModel> Commands { get; set; }
    }
}
